docs/static/ls-ls-config.asciidoc

```diff
@@ -0,0 +1,111 @@
+[[ls-to-ls]]
+=== Configuring Logstash-to-Logstash Communication
+
+You can set up communication between two Logstash machines by connecting the Lumberjack output to the Beats input. You may need this configuration if the data path crosses network or firewall boundaries, for example. If you don't have a compelling need for Logstash-to-Logstash communication, then don't implement it.
+
+[[ls-to-ls-overview]]
+==== Configuration overview
+
+Use the Lumberjack protocol to connect two Logstash machines.
+
+. Generate a trusted SSL certificate (required by the lumberjack protocol).
+. Save the SSL certificate on the upstream Logstash machine.
+. Save the SSL certificate and key on the downstream Logstash machine.
+. Set the downstream Logstash machine to listen for incoming Lumberjack connections through the Beats input.
+. Set the upstream Logstash machine to use the Lumberjack output to send data.
+. Test it.
+
+[[generate-self-signed-cert]]
+===== Generate a self-signed SSL certificate and key
+
+You can use the `openssl req` command to generate a self-signed certificate and key. The `openssl req` command is available on any operating system that comes with SSL.
+
+Run the following command:
+
+[source,shell]
+----
+openssl req -x509 -batch -nodes -newkey rsa:2048 -keyout lumberjack.key -out lumberjack.cert -subj /CN=localhost
+----
+
+where:
+
+* `lumberjack.key` specifies the name of the SSL key to be created
+* `lumberjack.cert` specifies the name of the SSL certificate to be created
+* `localhost` specifies the name of the upstream Logstash computer
+
+
+This command produces output similar to the following:
+
+[source,shell]
+----
+Generating a 2048 bit RSA private key
+.................................+++
+....................+++
+writing new private key to 'lumberjack.key'
+----
+
+[[save-cert-key]]
+===== Save the SSL certificate and key
+
+Copy the SSL certificate to the upstream Logstash machine.
+
+Copy the SSL certificate and key to the downstream Logstash machine.
+
+[[config-demo]]
+==== Demo
+
+You can demo the configuration by running two Logstash instances on one computer.
+
+[[save-cert-ls1]]
+===== Start the upstream Logstash instance
+
+Run the following command to start Logstash and generate test events:
+
+[source,shell]
+----
+bin/logstash -e 'input { generator { count => 5 } } output { lumberjack { codec => json hosts => "localhost" ssl_certificate => "lumberjack.cert" port => 5000 } }'
+----
+
+where:
+
+* `5` specifies the number of events to be generated
```

Comment:
If we do keep this, as a convention should we use `5` or `count => 5` or `count`? In other words, do we document the value or the key for that value? This is something we should decide as a style convention for our docs in general.

Reply:
@dedemorton I'd like to get your thoughts on the best approach. (I like to use a sample value and then explaining it rather than going too generic. At the same time, I want to be consistent with other writers and products.)

Reply:
@karenzone The page was not refreshed, so I didn't see your comment until after I wrote mine. Striving for consistency is a bit tricky atm because we have a lot of hands in the docs. I'd focus on making your content internally consistent, accurate, and easy-to-read, and not worry as much about other projects and writers.